utils/rocvideodecode/roc_video_dec.h

```diff
@@ -68,11 +68,11 @@ typedef enum OutputSurfaceMemoryType_enum {
 #define STR(X) std::string(X)
 
 #if DBGINFO
-#define INFO(X) std::clog << "[INF] " << " {" << __func__ <<"} " << " " << X << std::endl;
+#define ROCDEC_INFO(X) std::clog << "[INF] " << " {" << __func__ <<"} " << " " << X << std::endl;
 #else
-#define INFO(X) ;
+#define ROCDEC_INFO(X) ;
 #endif
-#define ERR(X) std::cerr << "[ERR] "  << " {" << __func__ <<"} " << " " << X << std::endl;
+#define ROCDEC_ERR(X) std::cerr << "[ERR] "  << " {" << __func__ <<"} " << " " << X << std::endl;
```

Comment:
please change `ERR` to `ROCDEC_ERR` and `INFO` to `ROCDEC_INFO` for all the tests. 

pre-checking tests are failing for few tests videodecodeperf and videodecodebatch 

![image](https://github.com/user-attachments/assets/f2ff5b97-652a-4f6e-8746-1547fb1f2759)

![image](https://github.com/user-attachments/assets/520b9018-1dce-42b1-81fa-11f3df6991a5)